deps.bzl

```diff
@@ -1580,3 +1580,9 @@ def install_buildbuddy_dependencies():
         sum = "h1:wFXVbFY8DY5/xOe1ECiWdKCzZlxgshcYVNkBHstARME=",
         version = "v0.1.2",
     )
+    go_repository(
```

Comment:
minor nit: since this is such a big CL, you might (next time) consider sending stuff like this separately beforehand.